wrappers/Julia/CoolProp.jl

```diff
@@ -55,6 +55,7 @@ end
 
 # CoolProp::get_parameter_information_string
 function get_parameter_information_string(Key::AbstractString)
+message_buffer[1:4] = UInt8[108 111 110 103] # "long" written in UInt8
```

Comment:
Then you could try to dump the buffer for debugging.
Actually, I still suspect not-null-terminating case: first, your buffer is created zero-filled, then you copy "long" there, after that, result is put there that is longer, and then you copy "long" (4 bytes) to buffer already filled with previous grabage.